kornia/geometry/boxes.py

```diff
@@ -432,10 +436,6 @@ def transform_boxes_(self, M: torch.Tensor) -> "Boxes":
         """Inplace version of :func:`Boxes.transform_boxes`"""
         return self.transform_boxes(M, inplace=True)
 
-    @property
-    def _is_batched(self) -> bool:
```

Comment:
I would keep `is_batched` property

Reply:
User do not need to care if it batched since it will always align with the input.